app/lib/constituency_api.rb

```diff
@@ -6,9 +6,25 @@ class Constituency < Struct.new(:name, :mp)
 
   class Mp < Struct.new(:id, :name, :start_date)
     URL = "http://www.parliament.uk/biographies/commons"
+
+    def initialize(id, name, start_date)
+      self.id = id.to_i
```

Comment:
I'm not sure we need to bother with converting this to an integer.  (I know we were before, but it's not super useful.)